tutorials/applications/Simulating Sequences with Errors and Noise.ipynb

```diff
@@ -0,0 +1,899 @@
+{
```

Comment:
<p>Just comments on the text.</p><p>I would say "During the imaging process" rather than "During the simulation" in the eps' description. </p><p>Not sure about what you mean by "Amplitude noise". I would use this term for the random variation of Omega and another for the fact that the waist of the laser is finite. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/161/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Yup, this is much clearer.</p>